bedrock/firefox/templates/firefox/home/index-master.html

```diff
@@ -0,0 +1,284 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/base/base-protocol.html" %}
+{% from "macros-protocol.html" import call_out, call_out_compact, hero, feature_card with context %}
+
+{# Bug 1438302 Avoid duplicate content for en-CA and en-GB pages. #}
+{% if LANG == 'en-CA' %}
+  {% set title_suffix = 'Firefox (CA)' %}
+{% elif LANG == 'en-GB' %}
+  {% set title_suffix = 'Firefox (UK)' %}
+{% else %}
+  {% set title_suffix = 'Firefox' %}
+{% endif %}
+{% block page_title %}{{ _('Protect your life online with Firefox and a family of privacy-first products') }}{% endblock %}
+{% block page_desc %}{{ _('Firefox is more than a browser. Learn more about Firefox products that handle your data with respect and are built for privacy anywhere you go online.') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/template/page-image-master.jpg') }}{% endblock %}
+{% block page_favicon %}{{ static('img/logos/master/favicon-64.png') }}{% endblock %}
+{% block page_favicon_large %}{{ static('img/logos/master/favicon-196.png') }}{% endblock %}
+{% block page_ios_icon %}{{ static('img/logos/master/favicon-180.png') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox-master') }}
+{% endblock %}
+
+{% block extrahead %}
+  <!--[if IE 9]>
+    {{ css_bundle('firefox-home-ie9') }}
+  <![endif]-->
+{% endblock %}
+
+{% block body_id %}firefox-master{% endblock %}
+{% block body_class %}{{ super() }} firefox-home{% endblock %}
+
+{% set referrals = '?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=firefox-home' %}
+
+{% if not is_translation %}
```

Comment:
I did this because I thought I was getting Teir 1 translation from the EU team and I'd have to implement them like the WNP 64. I've since found out that they can put the translations in Pontoon (I think? I haven't had direct confirmation of this just lots of circumstantial evidence). Should I change it?

Reply:
What you have works, so I wouldn't suggest needing to change it. Perhaps add a comment to explain why they were all grouped together? :)